sql/system_variables.h

```diff
@@ -228,6 +228,7 @@ struct System_variables {
   ulonglong histogram_generation_max_mem_size;
   ulong join_buff_size;
   ulonglong lock_wait_timeout_nsec;
+  ulonglong high_priority_lock_wait_timeout_nsec;
```

Comment:
I noticed the original patch wasn't updated this initializing this correctly, resulting in it being 0 unless it's explicitly set. The fix is to initialize it similar to lock_wait_timeout_nsec in mysqld.cc:get_options().